integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -209,10 +209,15 @@ case class CarbonLoadDataCommand(
     * 4. Session property CARBON_OPTIONS_SORT_SCOPE
     * 5. Default Sort Scope LOAD_SORT_SCOPE
     */
-    if (tableProperties.get(CarbonCommonConstants.SORT_COLUMNS) != null &&
-        tableProperties.get(CarbonCommonConstants.SORT_SCOPE) == null) {
-      // If there are Sort Columns given for the table and Sort Scope is not specified,
-      // we will take it as whichever sort scope given or LOCAL_SORT as default
+    if (tableProperties.get(CarbonCommonConstants.SORT_COLUMNS) == null ||
+        tableProperties.get(CarbonCommonConstants.SORT_COLUMNS).equals("")) {
```

Comment:
```suggestion
        tableProperties.get(CarbonCommonConstants.SORT_COLUMNS).trim.equals("")) {
```

Reply:
use `StringUtils.isBlank()` to avoid retrieve themember twice

Reply:
Done